v0/destinations/af/transform.js

```diff
@@ -89,7 +89,7 @@ function responseBuilderSimple(payload, message, destination) {
     updatedPayload.bundleIdentifier = bundleIdentifier;
   }
 
-  const sharingFilter = destination.Config.sharingFilter || "all";
+  const { sharingFilter } = destination.Config.sharingFilter;
```

Comment:
Did you mean to use this?
```suggestion
  const { sharingFilter } = destination.Config;
```

Reply:
yes